homeassistant/components/tado/repairs.py

```diff
@@ -0,0 +1,29 @@
+"""Repair implementations."""
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import issue_registry as ir
+
+from .const import CONST_OVERLAY_MANUAL, CONST_OVERLAY_TADO_DEFAULT, DOMAIN
+
+
+def manage_water_heater_fallback_issue(
+    hass: HomeAssistant,
+    water_heater_entities: list,
+    integration_overlay_fallback: str | None,
+) -> None:
+    """Notify users about water heater respecting fallback setting."""
+    if (
+        integration_overlay_fallback
+        in [CONST_OVERLAY_TADO_DEFAULT, CONST_OVERLAY_MANUAL]
+        and len(water_heater_entities) > 0
+    ):
+        for water_heater_entity in water_heater_entities:
+            ir.async_create_issue(
+                hass=hass,
+                domain=DOMAIN,
+                issue_id=f"water_heater_fallback_{water_heater_entity.zone_name}",
+                is_fixable=False,
+                is_persistent=True,
```

Comment:
Why is this persistent?

Reply:
The conditions for this repair issue can happen again but if issue is created once and ignored by the user we should not show again. The rationale for this issue to let users about the situation(If you have a text suggestion please tell me)
Using persistent was a suggestion from discord, in combination with issue id and this setting, issue is visible once per water heater zone even if the conditions happen again

Reply:
`is_persistent` should only be used for situation that aren't detectable upfront. For example, an automation that calls something deprecated only at runtime (so we can't know before it hits that code path).

In this case, the issue gets created on startup and should thus not be persistent.

Reply:
Okay, do you have a suggestion for me for storage. In this case I have store somewhere a key value pair once issue is created. I can check this key before creating the issue.

I feel like config entry is not the best place for this and I dont know a better place to store this info

Reply:
> Okay, do you have a suggestion for me for storage. 

There is nothing to store. It will create it, each time the issue is detected/occurs (on init of the integration).

```suggestion
                is_persistent=False,
```

Reply:
done :)